system-interface/src/instruction.rs

```diff
@@ -14,6 +14,13 @@
 //!
 //! [rent exempt]: https://solana.com/docs/core/accounts#rent-exemption
 //!
+//! The [`create_account`] function requires that the account have zero
+//! lamports. [`create_account_allow_prefund`] allows for the account to have
+//! lamports prefunded; note that without feature activation of SIMD-0312,
```

Comment:
Did you want the link later to reference here?
```suggestion
//! lamports prefunded; note that without feature activation of [SIMD-0312],
```